src/main/scala/is/hail/expr/AST.scala

```diff
@@ -511,7 +511,14 @@ case class ReferenceGenomeDependentFunction(posn: Position, fName: String, grNam
     case _ => FunctionRegistry.call(fName, args, args.map(_.`type`).toSeq, Some(rTyp))
   }
 
-  def toIR(agg: Option[String] = None): Option[IR] = None
+  def toIR(agg: Option[String] = None): Option[IR] = {
```

Comment:
if my improvement to toIR lands first you'll hit a rebase conflict. The big chance is that you'll need to change `anyFailAllFail` to `all` and wrap the IRFunctionRegistry.lookupConversion similarly to other places, but with a relevant error message.